pk3DS.Core/Randomizers/PersonalRandomizer.cs

```diff
@@ -67,56 +75,116 @@ public void Execute()
         {
             for (var i = 1; i < Table.Length; i++)
                 Randomize(Table[i], i);
+            
+            List<Action<PersonalInfo, PersonalInfo>> propagations = new List<Action<PersonalInfo, PersonalInfo>>();
```

Comment:
C# lets you use `var` for variable declarations, and the resulting variable will be whatever type is being assigned to it. It's great for things such as `var x = new Thing();`